backend/app/routes/search.py

```diff
@@ -5,33 +5,39 @@
 from fastapi import HTTPException, Query
 from httpx import AsyncClient
 
-from app.config import SOLR_URL
+from app.solr.operations import search
 
 from .util import DEFAULT_SORT, internal_api_router
 
 
+# pylint: disable=too-many-arguments
 @internal_api_router.get("/search", name="apis:get-search")
 async def search_get(
-    q: str = Query(..., description="Query string"),
     collection: str = Query(..., description="Collection"),
-    rows: int = Query(10, description="Row count", gte=0, le=100),
-    cursor: str = Query("*", description="Cursor"),
+    q: str = Query(..., description="Query string"),
```

Comment:
This same as above. Docstrings with params or URL to its documentations + examples